x11-misc/sddm/files/sddm.initd

```diff
@@ -0,0 +1,95 @@
+#!/sbin/openrc-run
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License, v2
+
+command="/usr/bin/sddm"
+command_background="yes"
+pidfile="/run/sddm.pid"
+retry="TERM/5/TERM/5"
+
+depend() {
+    need localmount elogind
```

Comment:
This can cause problems. On my system I hit
```
# /etc/init.d/sddm restart
 * Starting elogind ...
 * start-stop-daemon: /lib64/elogind/elogind is already running                                                                 [ !! ]
 * ERROR: elogind failed to start
 * ERROR: cannot start sddm as elogind would not start
```
That's because I elogind was started manually by some other services...